Xamarin.Forms.Core.UnitTests/StyleSheets/StyleTests.cs

```diff
@@ -60,7 +60,8 @@ public void StylesAreCascading()
 			Assume.That(style, Is.Not.Null);
 
 			var label = new Label();
-			var layout = new StackLayout {
+			var layout = new StackLayout
```

Comment:
don't change the style. thx

Reply:
reverted formatting changes that happened on save.

> please provide a better title and description
What do you mean by this? for the entire pull-request?